homeassistant/components/danfoss_air.py

```diff
@@ -0,0 +1,89 @@
+"""
+Support for Danfoss Air HRV.
+
+Configuration:
+    danfoss_air:
+        host: IP_OF_CCM_MODULE
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/danfoss_air/
+"""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['pydanfossair==0.0.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+DANFOSS_AIR_PLATFORMS = ['sensor', 'binary_sensor']
+DOMAIN = 'danfoss_air'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=60)
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+    }),
+}, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Set up the Danfoss Air component."""
+    conf = config[DOMAIN]
+
+    danfoss_config = {}
+    danfoss_config['host'] = conf[CONF_HOST]
```

Comment:
Why is this object needed?

Reply:
It is used as configuration for the DanfossClient class that is used in DanfossAir class. It is passed as constructor argument. I will move the construction of the dictionary to just before making an instance of DanfossClient. This makes it a bit more clear what it is actually used for.

Reply:
It was an attempt to make it easy to add additional configuration values. Thinking about it will increase the risk of errors in the future. So I have changed it to pass it directly to the constructor as you suggest.